rubocop-airbnb/config/rubocop-layout.yml

```diff
@@ -344,7 +343,6 @@ Layout/MultilineMethodCallIndentation:
   SupportedStyles:
   - aligned
   - indented
-  IndentationWidth:
```

Comment:
`IndentationWidth` isn't supported anymore, although I can't find in the rubocop changelog where exactly that happened.

Reply:
The name of the cop is `Layout/IndentationWidth`. This line with bare `IndentationWidth` should probably never have been here, so you were right to remove it.